lib/features/chat/domain/helix_images.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'helix_images.freezed.dart';
+
+@freezed
+class HelixImages with _$HelixImages {
```

Comment:
can be moved into helix_chat_emotes as there is only one class needing this now. Rename to `EmoteImageAssets` or something more descriptive.